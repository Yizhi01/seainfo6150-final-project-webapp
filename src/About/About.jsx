import React from "react";
import styles from "./About.module.css";

const About = () => {
  return (
    <div className={styles.aboutContainer}>
      <img className={styles.aboutImg} src="http://source.unsplash.com/ZrhtQyGFG6s/1000x619" alt="chat with friends"></img>
      <p>Hey Friends <span role="img" aria-label="wave hand">👋🏼 </span>, </p>
      <p>
        Are you looking for trying new recipes? Expecting of your holiday feast with a bang? Pressed for time?
      </p>
      <p>
        Welcome to Tasty Food website. It’s a good place to give you some inspiration when you try to cook at home and allow you to cook delicious dishes efficiently and effortlessly from anywhere in the world.
      </p>
      <p>Also, if you are a cook lover and like to share your own recipe, please <a href="/signUp">join our community</a>, submit your recipe <a href="/newRecipe">here</a>!</p>
    </div>
  )
}

export default About;