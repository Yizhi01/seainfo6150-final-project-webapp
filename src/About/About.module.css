.aboutContainer {
  margin: 30px 25%;
  display: flex;
  flex-direction: column;
  flex-wrap: wrap;
  justify-content: center;
}

.aboutImg {
  align-self: center;
  width: 100%;
}

.aboutContainer p {
  padding: 10px 0;
}

.aboutContainer p a {
  font-weight: 600;
  color: teal;
}

@media (max-width: 800px) {
  .aboutContainer {
    margin: 30px 10%;
  }
}