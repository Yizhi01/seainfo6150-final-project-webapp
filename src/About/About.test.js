import React from "react";
import { render } from "@testing-library/react";
import { BrowserRouter as Router } from "react-router-dom";
import About from "./About.jsx";

it("renders the About component correctly", () => {
  const { container } = render(
    <Router>
        <About />
    </Router>
  );
  expect(container).toMatchSnapshot();
});