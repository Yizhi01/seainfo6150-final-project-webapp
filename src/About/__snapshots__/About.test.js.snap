// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the About component correctly 1`] = `
<div>
  <div
    class="aboutContainer"
  >
    <img
      alt="chat with friends"
      class="aboutImg"
      src="https://source.unsplash.com/ZrhtQyGFG6s/1000x619"
    />
    <p>
      Hey Friends 
      <span
        aria-label="wave hand"
        role="img"
      >
        👋🏼 
      </span>
      , 
    </p>
    <p>
      Are you looking for trying new recipes? Expecting of your holiday feast with a bang? Pressed for time?
    </p>
    <p>
      Welcome to Tasty Food website. It’s a good place to give you some inspiration when you try to cook at home and allow you to cook delicious dishes efficiently and effortlessly from anywhere in the world.
    </p>
    <p>
      Also, if you are a cook lover and like to share your own recipe, please 
      <a
        href="/signUp"
      >
        join our community
      </a>
      , submit your recipe 
      <a
        href="/newRecipe"
      >
        here
      </a>
      !
    </p>
  </div>
</div>
`;
